src/main/java/org/sonarlint/intellij/ui/SonarLintChangedPanel.java

```diff
@@ -132,4 +146,16 @@ private void createTree() {
     tree = new IssueTree(project, model);
     tree.addTreeSelectionListener(e -> issueTreeSelectionChanged());
   }
+
+  @Override
+  public void dispose() {
+    changeListManager.removeChangeListListener(vcsChangeListener);
+  }
+
+  private ChangeListListener vcsChangeListener = new ChangeListAdapter() {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Make "vcsChangeListener" transient or serializable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1948)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1213)